kriswallsmith/buzz/0.17/config/packages/buzz.yaml

```diff
@@ -0,0 +1,27 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
+    # Clients
+    Buzz\Client\Curl:
+        arguments:
+            - timeout: 10
+
+    Buzz\Client\FileGetContents:
```

Comment:
should all these classes be available for autowiring ? If no, you should rather give these another name.

Reply:
Yeah. I think so. One may only be interested in using clients.

Reply:
but then, they should still typehint the client interface, not specific implementation. currently, the interface is not autowirable here, while implementations are.

Reply:
Fixed!